.github/workflows/erlang.yml

```diff
@@ -10,17 +10,21 @@ jobs:
 
     strategy:
       matrix:
-        otp: [21.3, 22.3, 23.0.4, 23.2.7.0, 24.0]
-    container:
-      image: erlang:${{ matrix.otp }}-alpine
+        otp: ['21.3', '22.3', '23.3']
+        rebar: ['3.15.2']
+        include:
+        - otp: '24.0'
```

Comment:
3.16.1 goes all the way down to 22, so the exception to the rule is actually 21.3/3.15.2 :)